clang/test/CIR/CodeGen/struct-init.c

```diff
@@ -0,0 +1,12 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-llvm %s -o %t.ll
+// RUN: FileCheck --input-file=%t.ll %s -check-prefix=LLVM
```

Comment:
Nice. Since you are not checking any CIR file, this test would be best to live inside `test/CIR/Lowering`

Reply:
@bcardosolopes  done!